astrobee/config/mobility/planner_qp.config

```diff
@@ -39,10 +39,10 @@ parameters = {
     id = "robot_radius",
     reconfigurable = true,
     type = "double",
-    default = 0.16,
+    default = 0.22,
     min = 0.0,
     max = 1.0,
-    description = "Radius of the robot considered in the planner (default is 0.15 * 2* sqrt(3))",
+    description = "Radius of the robot considered in the planner (default is 0.15 * sqrt(2))",
```

Comment:
The previous parameter was not enough, I changed it to 0.15*sqrt(2) to account for the diagonal

Reply:
As we just discussed in chat, to find the radius of the circumscribing sphere of a cube with width ~0.32, the formula would be 0.16 * sqrt(3) which comes to 0.28 (rounding up).